lib/load_wallet.dart

```diff
@@ -234,6 +235,29 @@ class LoadWalletScreen extends StatelessWidget {
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: [
+            const Spacer(),
+            Consumer<WalletState>(
+              builder: (context, walletState, child) {
+                return DropdownButton<String>(
+                  hint: const Text('Select a network'),
+                  value: walletState.network,
+                  onChanged: (String? newValue) {
+                    try {
+                      walletState.setNetwork(newValue!);
+                    } catch (e) {
+                      Logger().e("Failed to set network to $newValue: $e");
+                    }
+                  },
+                  items: <String>['main', 'signet', 'test', 'regtest']
```

Comment:
We should remove 'regtest' here. Also we should rename 'main' to 'mainnet' and 'test' to 'testnet'.

Reply:
Actually I was constrained by the fact I wanted to directly use the strings as arguments to set up the network, so I needed it to be exactly as in core arguments. But we can rework that too I guess.

Reply:
Ok I found a solution to that too